_posts/2020-10-01-intro-developer-advocate.md

```diff
@@ -0,0 +1,111 @@
+
+![](/assets/blog/developer-advocate/brian.jpg)
+
+<!--more-->
+
+My career in computation and databases started in a previous life in the
+military. As luck would have it, I worked on a Big data team as my first job
+out of college! I was in a Hive shop that dealt with the typical outdated
+runtime and slow query turnaround. Eventually, our architect introduced our team
+to Presto as an alternative, and I worked with him to start testing and moving
+our existing use cases built on Hive to use Presto. We also used Elasticsearch
+and had a few cases that needed to perform joins and unions over the datasets in
+both Elasticsearch and Hive. There were a few use cases that were not going to
+immediately be transferable without some modification to the Elasticsearch
+connector. The first was [adding support for the Elasticsearch array 
+types](https://github.com/prestosql/presto/issues/2441), and the second was, 
+[support for nested types](https://github.com/prestosql/presto/issues/754). 
+
+My first interaction with the Presto community was incredible! As a serial
+open-source attempter, I had always wanted to get invested in an open-source
+project. I had started pull requests in various projects. Sometimes I ran into 
+jerks, in other cases the rules were daunting or too confusing to start, or I
+created a pull request only to have it sit there with no communication
+as to why it wasn't accepted or even looked at. However, when I first joined the
+Slack channel I searched to see if there was already a discussion about array
+types in the history. I ran into [a discussion between Dain and Martin about
+this issue](https://prestosql.slack.com/archives/CP1MUNEUX/p1570064139005900). 
+I conversed with Martin, who was incredibly polite and willing to take time to 
+discuss how this should be implemented. When I actually pulled the code, I saw
+how well written and maintained it was compared to many open-source projects I
+had seen in the past. I made a few changes, wrote a test around my use case, and
+signed a CLA agreement. After a couple of weeks, my pull request was merged
+and I had finally contributed to open-source. After that interaction and seeing
+the code I wanted to do more. I really saw something special with this
+community.
+
+While many Presto contributors are doing amazing work contributing code, I
+noticed there were some holes in other areas of the community that needed to be
+filled. I started answering questions on Slack, LinkedIn, and Twitter and I
+planned out a Udemy course for Presto. The [initial 
+video](https://youtu.be/RPaG0Gu2I6c) I piloted was about how to tune memory in
+ Presto.
+
+Around this time I got into contact with some folks at Starburst about joining 
+them to work with the community and Presto full-time! As I joined we hadn't
+figured out what my exact role was at Starburst but eventually decided I would
+best serve as a Developer Advocate in Developer Relations. If you’re like 
```

Comment:
as a developer advocate. The aim of the role is exactly what I set out to do before I joined. As a developer advocate, ...